stdlib/public/runtime/Metadata.cpp

```diff
@@ -4207,6 +4301,110 @@ static bool doesNotRequireInstantiation(
   return true;
 }
 
+static const unsigned swift_ptrauth_key_associated_type =
+  swift_ptrauth_key_associated_type;
```

Comment:
I don't see any way to expand the context around my previous comment, so this is where the self-assignment is happening.